Create README.md

alex4200 opened the pull request:
Adding acknowledgement and copyright

alex4200 committed:
Create README.md

Adding acknowledgement and copyright

matz-e reviewed (commented):
This is a repository that just "mirrored" the official code. No work done from our side, really.

jamesgking commented:
My recollection from conversation with Felix in the Computing Division meeting is that we can populate the README with the explanation on the mirror. And thus there is no funding from BBP that can be attributed to the code itself. I will update the PR with what I think would be a good description

jamesgking committed:
Update README.md

Adapted text to reflect why this repository exists to contain code that was not developed at BBP. Removed the funding text since no funds were used to write any of the code contained in this library.

jamesgking commented:
I have updated the text to reflect our use case & history. Open to additional suggestions
